awx/ui/src/screens/InstanceGroup/Instances/InstanceListItem.js

```diff
@@ -108,7 +108,7 @@ function InstanceListItem({
             rowIndex,
             isSelected,
             onSelect,
-            disable: false,
+            disable: instance.node_type === 'control',
```

Comment:
I take it that this is the list of instances that are members of a particular instance group?  Do we have an equivalent screen file for the list of instance groups that an instance is a member of?  In doing the back-end code, I realized that both needed to be dealt with.